src/common/options/rgw.yaml.in

```diff
@@ -3245,6 +3245,38 @@ options:
   - rados
   - dbstore
   - motr
+- name: motr_profile_fid
```

Comment:
maybe this is ok;  let's discuss in refactoring call

Reply:
@soumyakoduri, could you elaborate a bit - why not the right place? What would be the possible alternative?